app/Http/Controllers/HomeController.php

```diff
@@ -13,4 +15,9 @@ public function rules()
     {
         return view('rules');
     }
+
+    public function redirectToPaste()
+    {
+        return Redirect::to('https://paste.laravel.io');
```

Comment:
Sure! I was looking in other files and the first I found used `Validator::` Facade, so I used that one (/cc @introwit ). Let me update and adhere to your requests 👍